pkg/provider/resource_fareplica.go

```diff
@@ -478,7 +513,7 @@ func (r *FAReplicaResource) read(ctx context.Context, fAReplicaResourceModel *FA
 	fAReplicaResourceModel.MetricsUrl = apiCluster.MetricsUrl
 	fAReplicaResourceModel.BackupRetentionPeriod = types.StringPointerValue(apiCluster.BackupRetentionPeriod)
 	fAReplicaResourceModel.PrivateNetworking = types.BoolPointerValue(apiCluster.PrivateNetworking)
-
+	fAReplicaResourceModel.PgIdentity = types.StringValue(*apiCluster.PgIdentity)
```

Comment:
may get a nil error here, as it is only applicable if the cluster is TDE enabled and key is set